tests/secrets.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+    rm -rf $TESTSDIR/containers
+    rm -rf $TESTSDIR/rhel
+    mkdir $TESTSDIR/containers
+    touch $TESTSDIR/mounts.conf
+    MOUNTS_PATH=$TESTSDIR/containers/mounts.conf
+    echo "$TESTSDIR/rhel/secrets:/run/secrets" > $MOUNTS_PATH
+
+    mkdir $TESTSDIR/rhel
+    mkdir $TESTSDIR/rhel/secrets
+    touch $TESTSDIR/rhel/secrets/test.txt
+    echo "Testing secrets mounts. I am mounted!" > $TESTSDIR/rhel/secrets/test.txt
+}
+
+@test "bind secrets mounts to container" {
+    if ! which runc ; then
+		skip
+	fi
+	runc --version
+	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
```

Comment:
The `bulidah` helper function that runs the `buildah` binary runs it with debugging enabled, so you probably want to add `--debug=false` to undo that, when the output is being captured for use this way.

Reply:
fixed.